pkg/util/restoptions/configgetter.go

```diff
@@ -108,7 +107,7 @@ func (g *configRESTOptionsGetter) GetRESTOptions(resource unversioned.GroupResou
 		// TODO: choose destination group/version based on input group/resource
 		// TODO: Tune the cache size
 		groupVersion := unversioned.GroupVersion{Group: "", Version: g.masterOptions.EtcdStorageConfig.OpenShiftStorageVersion}
-		g.etcdHelper = etcdstorage.NewEtcdStorage(etcdClient, kapi.Codecs.LegacyCodec(groupVersion), g.masterOptions.EtcdStorageConfig.OpenShiftStoragePrefix, false, genericserveroptions.DefaultDeserializationCacheSize)
+		g.etcdHelper = etcdstorage.NewEtcdStorage(etcdClient, kapi.Codecs.LegacyCodec(groupVersion), g.masterOptions.EtcdStorageConfig.OpenShiftStoragePrefix, false, 1000)
```

Comment:
Agree

On Thu, Sep 1, 2016 at 1:39 PM, Andy Goldstein <notifications@github.com>
wrote:

> In pkg/util/restoptions/configgetter.go
> <https://github.com/openshift/origin/pull/10719#discussion_r77219333>:
>
> > @@ -108,7 +107,7 @@ func (g *configRESTOptionsGetter) GetRESTOptions(resource unversioned.GroupResou
> >  		// TODO: choose destination group/version based on input group/resource
> >  		// TODO: Tune the cache size
> >  		groupVersion := unversioned.GroupVersion{Group: "", Version: g.masterOptions.EtcdStorageConfig.OpenShiftStorageVersion}
> > -		g.etcdHelper = etcdstorage.NewEtcdStorage(etcdClient, kapi.Codecs.LegacyCodec(groupVersion), g.masterOptions.EtcdStorageConfig.OpenShiftStoragePrefix, false, genericserveroptions.DefaultDeserializationCacheSize)
> > +		g.etcdHelper = etcdstorage.NewEtcdStorage(etcdClient, kapi.Codecs.LegacyCodec(groupVersion), g.masterOptions.EtcdStorageConfig.OpenShiftStoragePrefix, false, 1000)
>
> If changing the cache to be per resource requires that we modify idle
> connections per helper per resource, I want to wait for 1.3.1 for this.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/10719/files/5787df6f68b4ae442b248e9d0623fbda04c71562#r77219333>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABG_pzwrcOMwf558BnEbxZmisCtFEm48ks5qlw3TgaJpZM4Jw21S>
> .
>